src/models/pmv_ppd.js

```diff
@@ -0,0 +1,486 @@
+ * Please read more in the Note below.
+ *
+ * Notes:
+ * You can use this function to calculate the `PMV` and `PPD` in accordance with
+ * either the ASHRAE 55 2020 Standard {@link #ref_1|[1]} or the ISO 7730 Standard {@link #ref_2|[2]}.
+ *
+ * _PMV:
+ * {@link https://en.wikipedia.org/wiki/Thermal_comfort#PMV/PPD_method|PMV}
+ * _PPD:
+ * {@link https://en.wikipedia.org/wiki/Thermal_comfort#PMV/PPD_method|PPD}
+ * _Addendum C to Standard 55-2020:
+ * {@link https://www.ashrae.org/file%20library/technical%20resources/standards%20and%20guidelines/standards%20addenda/55_2020_c_20210430.pdf|Addendum C to Standard 55-2020}
+ *
+ * This is a version that supports scalar arguments.
+ * @see {@link pmv_ppd_array} for a version that supports arrays.
+ *
+ * @public
+ * @memberof models
+ * @docname Predicted Mean Vote (PMV) and Predicted Percentage of Dissatisfied (PPD)
+ *
+ * @param { number } tdb - dry bulb air temperature, default in [°C] in [°F] if `units` = 'IP'
+ * @param { number } tr - mean radiant temperature, default in [°C] in [°F] if `units` = 'IP'
+ * @param { number } vr - relative air speed, default in [m/s] in [fps] if `units` = 'IP'
+ * Note: vr is the relative air speed caused by body movement and not the air
+ * speed measured by the air speed sensor. The relative air speed is the sum of the
+ * average air speed measured by the sensor plus the activity-generated air speed
+ * (Vag). Where Vag is the activity-generated air speed caused by motion of
+ * individual body parts. vr can be calculated using the function `v_relative` which is in .utilities.js.
+ * @param { number } rh - relative humidity, [%]
+ * @param { number } met - metabolic rate
+ * @param { number } clo - clothing insulation
+ * Note: The activity as well as the air speed modify the insulation characteristics
+ * of the clothing and the adjacent air layer. Consequently, the ISO 7730 states that
+ * the clothing insulation shall be corrected {@link #ref_2|[2]}. The ASHRAE 55 Standard corrects
+ * for the effect of the body movement for met equal or higher than 1.2 met using
+ * the equation clo = Icl × (0.6 + 0.4/met) The dynamic clothing insulation, clo,
+ * can be calculated using the function `clo_dynamic` which is in .utilities.js.
+ * @param { number } [wme=0] - external work, default is 0
+ * @param { "ISO"|"ASHRAE" } [standard="ISO"] - comfort standard used for calculation
+ * - If "ISO", then the ISO Equation is used
+ * - If "ASHRAE", then the ASHRAE Equation is used
+ * Note: While the PMV equation is the same for both the ISO and ASHRAE standards,
+ * the ASHRAE Standard Use of the PMV model is limited to air speeds below 0.10m/s (20 fpm).
+ * When air speeds exceed 0.10 m/s (20 fpm), the comfort zone boundaries are adjusted based on the SET model.
+ * This change was introduced by the `Addendum C to Standard 55-2020`_
+ * @param { Pmv_ppdKwargs }kwargs - additional arguments
+ *
+ * @returns { Object } - Result of pmv and ppd
```

Comment:
I don't think this works, you have to define a `@typedef` type and return that with the two properties below

Reply:
done